schema-registry-mock-junit5/src/test/java/com/bakdata/schemaregistrymock/junit5/ProtobufRegistryMockExtensionTest.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2023 bakdata GmbH
```

Comment:
```suggestion
 * Copyright (c) 2024 bakdata GmbH
```

Also update other files (or let IntelliJ take care)